src/test/java/uk/gov/hmcts/reform/laubackend/cases/controllers/AccessRequestControllerTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.laubackend.cases.controllers;
+
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.reform.laubackend.cases.constants.AccessRequestType;
+import uk.gov.hmcts.reform.laubackend.cases.dto.AccessRequestLog;
+import uk.gov.hmcts.reform.laubackend.cases.request.AccessRequestPostRequest;
+import uk.gov.hmcts.reform.laubackend.cases.response.AccessRequestPostResponse;
+import uk.gov.hmcts.reform.laubackend.cases.service.AccessRequestService;
+
+import java.time.format.DateTimeParseException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class AccessRequestControllerTest {
+
+    @Mock
+    private AccessRequestService accessRequestService;
+
+    @InjectMocks
+    private AccessRequestController accessRequestController;
+
```

Comment:
There is no test for missing any parameter.

Reply:
It is because I don't do any validation of parameters, all done by the library, and it is not in the controller. That testing is done in integration as it is better place to have all machinery involved. After your comment, I've added unit test for `AccessRequestLog.java`, which basically tests that required validation annotations are present.